packages/mui-material/src/Dialog/Dialog.test.js

```diff
@@ -318,6 +318,35 @@ describe('<Dialog />', () => {
       );
       expect(getByTestId('paper')).not.to.have.class(classes.paperFullScreen);
     });
+
+    it('scrolls if overflown on the Y axis', function test() {
+      if (/jsdom/.test(window.navigator.userAgent)) {
+        this.skip();
+      }
+
+      const ITEM_HEIGHT = 100;
+      const ITEM_COUNT = 10;
+
+      const { getByTestId } = render(
+        <Dialog
+          open
+          fullScreen
+          PaperProps={{ 'data-testid': 'paper', sx: { height: ITEM_HEIGHT } }}
+        >
+          {Array.from(Array(ITEM_COUNT).keys()).map((item) => (
+            <div key={item} style={{ flexShrink: 0, height: ITEM_HEIGHT }}>
+              {item}
+            </div>
+          ))}
+        </Dialog>,
+      );
+      const paperElement = getByTestId('paper');
```

Comment:
For the future, it should be

```suggestion
      const paperElement = screen.getByTestId('paper');
```

per point 2 of https://github.com/mui/mui-public/issues/173.